docs/install-w-machine.md

```diff
@@ -106,7 +106,7 @@ Here, you connect to each of the hosts and create a Swarm manager or node.
         manager   *        virtualbox   Running   tcp://192.168.99.100:2376           v1.9.1
 
 
-2. Your client should still be pointing to Docker Engine on `manager`. Use the following syntax to run a Swarm container as the primary Swarm manager on `manager`.
+2. Your client should still be pointing to Docker Engine on `manager`. Use the following syntax to run a Swarm container as the primary manager on `manager`.
```

Comment:
We are running a Swarm manager container here. 

IMHO we need to explicitly saying that we are going to run Swarm manager or Swarm agent as Swarm has 2 roles in a single binary.